app/routes/events/all-events/partials/featured-events.partial.tsx

```diff
@@ -47,12 +54,57 @@ const FeaturedEventsHits = () => {
 
   return (
     <>
-      <FeaturedEventCard card={firstHit} />
-      <div className="mt-24 grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 place-items-center md:place-items-start">
+      <div className="content-padding">
+        <FeaturedEventCard card={firstHit} />
+      </div>
+
+      {/* Desktop Layout */}
+      <div className="hidden mt-16 lg:mt-24 md:grid grid-cols-2 lg:grid-cols-3 gap-4 place-items-center md:place-items-start content-padding">
         {remainingHits?.map((hit) => (
           <OtherFeatureEventCardHit hit={hit} key={hit.objectID} />
         ))}
       </div>
+
+      {/* Mobile Layout - Carousel */}
```

Comment:
Can you fix the filter buttons being cut off on the side on mobile?

<img width="413" height="376" alt="Image" src="https://github.com/user-attachments/assets/5f720e94-a0b0-437a-bfe2-306f6ff5ed01" />